clients/python/src/kaskada/query.py

```diff
@@ -11,7 +11,7 @@
 import kaskada.formatters
 import kaskada.kaskada.v1alpha.destinations_pb2 as destinations_pb
 import kaskada.kaskada.v1alpha.query_service_pb2 as query_pb
-from kaskada.client import KASKADA_DEFAULT_SLICE, Client, get_client
+from kaskada.client import Client, get_client
```

Comment:
This was the big issue for the slice configuration.  This line was changed previously in a [refactor](https://github.com/kaskada-ai/kaskada/commit/18048724a03d7c633e1b132c42a0b5764ea204d4) that altered the logic. We now have a test in place to prevent this from happening.

Although the change is subtle: `from kaskada.client import KASKADA_DEFAULT_SLICE, Client, get_client` is evaluated the moment this module is imported and never again. 

`slice_filter = kaskada.client.KASKADA_DEFAULT_SLICE` means now we will always reach out the `kaskada.client` module and ask what is the latest value.